library/crates/Cargo.toml

```diff
@@ -7,7 +7,9 @@ path = "" # ignored by cargo raze
 
 [dependencies]
 alcoholic_jwt = "=4091.0.0"
+async-trait = "0.1.59"
 axum = { version = "=0.5.15", features = ["ws"] }
+axum-server = { version = "=0.4.4", features = ["tls-rustls"]}
```

Comment:
Add TLS support for the Axum HTTP server, as TypeDB Cloud must communicate with the user securely.